<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>442</width>
    <height>240</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>442</width>
     <height>17</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuActions">
    <property name="title">
     <string>Actions</string>
    </property>
    <addaction name="actionPlay"/>
    <addaction name="actionStop"/>
    <addaction name="actionPrev"/>
    <addaction name="actionNext"/>
    <addaction name="actionMute"/>
   </widget>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionAdd_to_Playlist"/>
    <addaction name="actionClear_Playlist"/>
   </widget>
   <widget class="QMenu" name="menuScreen">
    <property name="title">
     <string>Screen</string>
    </property>
    <addaction name="action16_9"/>
    <addaction name="action5_4"/>
   </widget>
   <widget class="QMenu" name="menuPlayback_Speed">
    <property name="title">
     <string>Playback Speed</string>
    </property>
    <addaction name="action1_0x"/>
    <addaction name="action1_5x"/>
    <addaction name="action2_0x"/>
    <addaction name="action5_0x"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuActions"/>
   <addaction name="menuScreen"/>
   <addaction name="menuPlayback_Speed"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>BottomToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionPlay"/>
   <addaction name="actionStop"/>
   <addaction name="actionPrev"/>
   <addaction name="actionNext"/>
   <addaction name="separator"/>
   <addaction name="actionHide"/>
   <addaction name="actionMute"/>
   <addaction name="actionStar1"/>
   <addaction name="actionStar2"/>
   <addaction name="actionStar3"/>
   <addaction name="actionStar4"/>
   <addaction name="actionStar5"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/icons/1473256486_BT_folder_file_open.png</normaloff>:/images/icons/1473256486_BT_folder_file_open.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
   <property name="toolTip">
    <string>Open a File</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionPlay">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/icons/play-button.png</normaloff>:/images/icons/play-button.png</iconset>
   </property>
   <property name="text">
    <string>Play</string>
   </property>
   <property name="toolTip">
    <string>Play Video</string>
   </property>
   <property name="shortcut">
    <string>Space</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/icons/stop.png</normaloff>:/images/icons/stop.png</iconset>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
   <property name="toolTip">
    <string>Stop Video</string>
   </property>
   <property name="shortcut">
    <string>S</string>
   </property>
  </action>
  <action name="actionAdd_to_Playlist">
   <property name="text">
    <string>Add to Playlist</string>
   </property>
  </action>
  <action name="actionHide">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Hide Playlist</string>
   </property>
   <property name="shortcut">
    <string>H</string>
   </property>
  </action>
  <action name="actionNext">
   <property name="text">
    <string>Next</string>
   </property>
   <property name="shortcut">
    <string>Media Next, Media Next</string>
   </property>
  </action>
  <action name="actionPrev">
   <property name="text">
    <string>Prev.</string>
   </property>
   <property name="shortcut">
    <string>Media Previous, Media Previous</string>
   </property>
  </action>
  <action name="actionClear_Playlist">
   <property name="text">
    <string>Clear Playlist</string>
   </property>
  </action>
  <action name="actionMute">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Mute</string>
   </property>
   <property name="shortcut">
    <string>M</string>
   </property>
  </action>
  <action name="actionFullscreen">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Fullscreen</string>
   </property>
  </action>
  <action name="action16_9">
   <property name="text">
    <string>Original</string>
   </property>
  </action>
  <action name="action5_4">
   <property name="text">
    <string>Stretch</string>
   </property>
  </action>
  <action name="actionStar5">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/ic_star_border_black_48dp_2x.png</normaloff>:/ic_star_border_black_48dp_2x.png</iconset>
   </property>
   <property name="text">
    <string>5 STARS</string>
   </property>
  </action>
  <action name="actionStar4">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/ic_star_border_black_48dp_2x.png</normaloff>:/ic_star_border_black_48dp_2x.png</iconset>
   </property>
   <property name="text">
    <string>4 STARS</string>
   </property>
  </action>
  <action name="actionStar3">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/ic_star_border_black_48dp_2x.png</normaloff>:/ic_star_border_black_48dp_2x.png</iconset>
   </property>
   <property name="text">
    <string>3 STARS</string>
   </property>
  </action>
  <action name="actionStar2">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/ic_star_border_black_48dp_2x.png</normaloff>:/ic_star_border_black_48dp_2x.png</iconset>
   </property>
   <property name="text">
    <string>2 STARS</string>
   </property>
  </action>
  <action name="actionStar1">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/ic_star_border_black_48dp_2x.png</normaloff>:/ic_star_border_black_48dp_2x.png</iconset>
   </property>
   <property name="text">
    <string>1 STAR</string>
   </property>
  </action>
  <action name="action1_0x">
   <property name="text">
    <string>1.0x</string>
   </property>
  </action>
  <action name="action1_5x">
   <property name="text">
    <string>1.5x</string>
   </property>
  </action>
  <action name="action2_0x">
   <property name="text">
    <string>2.0x</string>
   </property>
  </action>
  <action name="action5_0x">
   <property name="text">
    <string>5.0x</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="icons.qrc"/>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
